src/style.css

```diff
@@ -1,3 +1,90 @@
 body {
   margin: 0;
 }
+
+header {
+  font-size: 12px;
+  text-transform: uppercase;
+  box-sizing: border-box;
+  width: 100vw;
+  background-color: #ffff;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  padding: 0 50px;
+}
+
+.logo-link {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+ul {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.page-link {
+  margin-left: 20px;
+  position: relative;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  text-decoration: none;
+  color: #000;
+}
+
+.page-link:first-child {
+  margin-left: 0;
+}
+
+.page-link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: '';
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  width: 37px;
+  height: 4px;
+  border-radius: 8px;
+  background-color: #00acdc;
+}
+
+/* #REGION start */
+
+.roboto-regular {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+}
+
+.roboto-medium {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
+.roboto-bold {
+  font-family: Roboto, sans-serif;
+  font-weight: 700;
+  font-style: normal;
+}
```

Comment:
This part looks redundant. Just connect font-family and font-style to html selector and use appropriate font-weight in other selectors